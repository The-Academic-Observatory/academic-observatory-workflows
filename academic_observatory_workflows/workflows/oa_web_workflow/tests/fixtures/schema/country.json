version https://git-lfs.github.com/spec/v1
oid sha256:cbf95d4740b965d81cc4be9b3e84850bf106bab563a5666ebb0197a904c7016e
size 35416
