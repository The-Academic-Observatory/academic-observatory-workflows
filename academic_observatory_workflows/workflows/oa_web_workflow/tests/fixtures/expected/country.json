version https://git-lfs.github.com/spec/v1
oid sha256:5dc481955662d278d1a52b0bec1783fdee4cca3add6195d6535561d8ca98e307
size 83592
